README.md

```diff
@@ -126,6 +187,12 @@ excluded because for:
 ## Link checking as part of Github workflow
 
 We configured a [Github workflow](../.github/workflows/check-links.yml) to build the Gatsby site in
-production as a docker image and push it to Docker hub at [`jimmychu0807/substrate-docs`](https://hub.docker.com/repository/docker/jimmychu0807/substrate-docs).
+production as a docker image and push it to Docker hub at
+[`jimmychu0807/substrate-docs`](https://hub.docker.com/repository/docker/jimmychu0807/substrate-docs).
 The image is then launched as a service in the next CICD job. `checklinks:v3` is run against the
 running gatsby site in the docker container to check all links.
+
+## License
+
+TBD
+<!-- Substrate **documentation** is licence under the [Apache 2 license](./LICENSE). -->
```

Comment:
we need to set a proper license. Is, like the old devhub, this site publishable under Apache 2?